contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -410,33 +418,61 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, IERC677Rece
     emit SubscriptionConsumerAdded(subscriptionId, consumer);
   }
 
-  // @inheritdoc IFunctionsSubscriptions
-  function cancelSubscription(uint64 subscriptionId, address to) external override {
-    _whenNotPaused();
-    _onlySubscriptionOwner(subscriptionId);
-    _onlySenderThatAcceptedToS();
-
-    if (pendingRequestExists(subscriptionId)) {
-      revert CannotRemoveWithPendingRequests();
-    }
-
-    _cancelSubscriptionHelper(subscriptionId, to);
-  }
+  // Overriden in FunctionsRouter.sol
+  function _getSubscriptionDepositDetails() internal virtual returns (uint16, uint72);
 
-  function _cancelSubscriptionHelper(uint64 subscriptionId, address to) private {
+  function _cancelSubscriptionHelper(uint64 subscriptionId, address toAddress, bool skipSubscriptionDeposit) private {
```

Comment:
"skipSubscriptionDeposit" is a bit confusing. Maybe "checkDepositRefundability"?

Reply:
Sounds good, done